### DISCUSSION TITLE
Tailwind V3 Not Remove Unused Css

### DISCUSSION THREAD

**Discussion by HakanSungur** [tailwindlabs/tailwindcss#13643] [Help]
How can i remove unused css at V3? I have read all the documentation, but when I try what is explained there, it does not delete unused css.

**Comment by abhishek-parentune** [comment]
same question

**Comment by Arunjenson** [comment]
any update on this

**Comment by HakanSungur** [comment]
no yet.

Arun Jenson ***@***.***>, 22 Tem 2024 Pzt, 11:19 tarihinde
şunu yazdı:

**Comment by nielsbom** [comment]
The second time you asked this question it was better formulated: #13668

Shall we close this?
